cli/src/accounts/mod.rs

```diff
@@ -92,44 +122,177 @@ pub async fn create_aze_game_account(player_account_ids: Vec<u64>, small_blind:
     let key_pair = SecretKey::new();
     let pub_key: PublicKey = key_pair.public_key();
     let auth_scheme: AuthScheme = AuthScheme::RpoFalcon512 { pub_key };
 
     // initial seed to create the wallet account
     let init_seed: [u8; 32] = [
-        95, 113, 209, 94, 84, 105, 250, 242, 223, 203, 216, 124, 22, 159, 14, 132, 215, 85, 183, 204,
-        149, 90, 166, 68, 100, 73, 106, 168, 125, 237, 138, 16,
+        95, 113, 209, 94, 84, 105, 250, 242, 223, 203, 216, 124, 22, 159, 14, 132, 215, 85, 183,
+        204, 149, 90, 166, 68, 100, 73, 106, 168, 125, 237, 138, 16,
     ];
 
-    let (player_account, _) = create_basic_aze_player_account(
+    let (player_account, seed) = create_basic_aze_player_account(
         init_seed,
         auth_scheme,
-        AccountType::RegularAccountImmutableCode
-    ).unwrap();
+        AccountType::RegularAccountImmutableCode,
+    )
+    .unwrap();
+
+    let mut client: AzeClient = create_aze_client();
+    client.insert_account(
+        &player_account,
+        Some(seed),
+        &AuthSecretKey::RpoFalcon512(key_pair),
+    );
 
     Ok(player_account.id())
+}
+
+pub async fn consume_game_notes(account_id: AccountId) {
+    let mut client: AzeClient = create_aze_client();
+    client.sync_state().await.unwrap();
+    let account = client.get_account(account_id).unwrap();
+    let consumable_notes = client.get_consumable_notes(Some(account_id)).unwrap();
+    println!("Consumable notes: {:?}", consumable_notes.len());
+
+    for consumable_note in consumable_notes {
+        let tx_template = TransactionTemplate::ConsumeNotes(account_id, vec![consumable_note.note.id()]);
+        let tx_request = client.build_transaction_request(tx_template).unwrap();
+        execute_tx_and_sync(&mut client, tx_request).await;
+        println!("Waiting...");
+        sleep(Duration::from_secs(5)).await;
+    }
+}
+
+pub async fn enc_action(account_id: AccountId) {
+    // check account storage for a slot
+    // if slot == 1, mask. if slot == 2/3/4, remask
+
+    // 2 mock cards
+    let mut card_points = vec![]; // 2 cards for now
+    card_points.push(keygen(11)); // for now 
+    card_points.push(keygen(12));
+    let pub_key_agg = keygen(123); // for now
+    let masking_factor: u32 = 123;// for now
+
+    if account_id == AccountId::try_from(1085128954612151006).unwrap() { // for now
+        // Mask
+        let cipher_card_1 = mask(pub_key_agg, card_points[0], masking_factor);
+        println!("Cipher card 1 --> {:?}\n", cipher_card_1);
+        let cipher_card_2 = mask(pub_key_agg, card_points[1], masking_factor);
+        println!("Cipher card 2 --> {:?}", cipher_card_2);
+        return
+    }
+
+    // 2 mock masked cards
+    let cipher_card_1 = mask(pub_key_agg, card_points[0], masking_factor);
+    let cipher_card_2 = mask(pub_key_agg, card_points[1], masking_factor);
+
+    // remask
+    let remasked_card_1 = remask(pub_key_agg, cipher_card_1, masking_factor);
```

Comment:
We will be conditionalizing this right when to `mask` and when to `unmask`

Reply:
yes